src/main/java/lotto/view/LottoInput.java

```diff
@@ -0,0 +1,45 @@
+package lotto.view;
+
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class LottoInput {
+
+    private static final Pattern NUMBER_PATTERN = Pattern.compile("\\d+");
+    private static final int LIMIT_AMOUNT = 100000;
+    private static Scanner scanner = new Scanner(System.in);
+
+    public int getPurchasedAmount() {
+        System.out.println("구매금액을 입력하세요.");
+        String amount = scanner.nextLine();
+        return purchaseValid(amount);
+    }
+
+    public String getWinningNumbers() {
+        System.out.println("지난 주 당첨번호를 입력하세요.");
+        return scanner.nextLine();
+    }
```

Comment:
이 부분에서 로또 번호를 파싱해서 돌려준다면, 도메인 객체들이 파싱할 책임을 덜어낼 수 있다고 생각해요
```suggestion
    public List<Integer> getWinningNumbers() {
        System.out.println("지난 주 당첨번호를 입력하세요.");
        return parse(scanner.nextLine());
    }
```

Reply:
> 궁금한게 어쨋든 파싱하는것도 하나의 로직이라고 생각이 드는데 입력을 받는 클래스는 입력에 대한 것만 책임을 지게 한다고 생각해서 도메인으로 파싱하는것을 도메인으로 옮겼는데 입력쪽에 있는게 더 자연스러울까요?

파싱 로직도 하나의 비즈니스 로직이긴 하지만, 입력 형식은 외부 환경(사용자 입력 방식)에 따라 쉽게 변할 수 있어요. 
만약 입력 요구사항이 `,` 대신 `-`로 변경된다면, 이건 입력 방식의 변경이지 도메인의 핵심 로직 변경은 아닐 수 있습니다.

도메인 영역은 외부 변화에 최소한으로 영향을 받아야 합니다.
입력 형식에 따라 도메인이 자주 수정된다면, 도메인이 외부에 너무 의존적이라는 신호일 수 있습니다. 
그래서 입력에 관련된 처리는 도메인보다는 view 영역에서 담당하는 것이 적절할 것 같아요